packages/fxa-settings/src/components/Settings/FlowContainer/index.tsx

```diff
@@ -31,21 +31,21 @@ export const FlowContainer = ({
     : ftlMsgResolver.getMsg('flow-container-back', 'Back');
   return (
     <div
-      className="max-w-lg mx-auto mt-6 p-10 tablet:my-10 flex flex-col items-start bg-white shadow tablet:rounded-xl"
+      className="max-w-lg mx-auto mt-6 p-10 mobileLandscape:my-10 flex flex-col items-start bg-white shadow mobileLandscape:rounded-xl"
       data-testid="flow-container"
     >
       <Head title={title} />
 
-      <div className="flex items-center">
+      <div className="relative flex items-center">
         <button
           onClick={onBackButtonClick}
           data-testid="flow-container-back-btn"
           title={backButtonTitle}
-          className="relative w-8 h-8 -ms-2 me-2 tablet:me-10 tablet:-ms-18"
+          className="me-4 mobileLandscape:absolute mobileLandscape:p-4 mobileLandscape:-start-24"
```

Comment:
I think you can take `me-4` off as well. 😏 

With these changes I think the parent div `flex items-center` just scoots the arrow down slightly. Not sure if it'd be nicer to take those off and have more obvious control of the arrow with `top` in case we need to tweak later, or leave as-is. Nice improvement either way though, you'll be closing https://mozilla-hub.atlassian.net/browse/FXA-2526 inadvertently here too!

Reply:
The spacing is needed for small screens, where we only want spacing between the arrow and the heading. `me-4` adds spacing either to the left or right of the button depending on LTR/RTL